core/main/src/Core/Particle.ts

```diff
@@ -106,6 +108,7 @@ export class Particle implements IParticle {
 
         const shapeType = particlesOptions.shape.type;
         const reduceDuplicates = particlesOptions.reduceDuplicates;
+        this.orbitOptions = particlesOptions.orbit;
```

Comment:
This should not be done, the object is referenced instead of assigning the needed values. This can cause issues when reloading options or changing values manually.

Reply:
Removed.

Reply:
More details [here](https://github.com/matteobruni/tsparticles/pull/986#discussion_r513235768)